cmd/fireeye2nvd/fireeye2nvd.go

```diff
@@ -52,7 +51,7 @@ func init() {
 func main() {
 	baseURL := flag.String("base_url", baseURL, "FireEye API base URL")
 	sinceDuration := flag.String("since", "", "Golang duration string, use this instead of sinceUnix")
```

Comment:
s/use this instad of sinceUnix/overrides -since_unix flag/ maybe? Otherwise it is not very clear how these two interoperate.